src/ds3.c

```diff
@@ -805,18 +826,18 @@ static ds3_error* _net_process_request(const ds3_client* client, const ds3_reque
             date_header = g_strconcat("Date: ", date, NULL);
             headers = NULL;
 
-            if (request->md5 == NULL) {
-                md5_value = "";
-            } else {
-                char* md5_header;
-                md5_value = request->md5->value;
-                md5_header = g_strconcat("Content-MD5:", md5_value, NULL);
-                headers = curl_slist_append(headers, md5_header);
-                g_free(md5_header);
-            }
+            if (request->checksum == NULL) {
+	        checksum_value = "";
```

Comment:
I guess so? Looking at the code in the editor, the indentation shows up correctly. I'll fix that.